README.md

```diff
@@ -0,0 +1,175 @@
+
+  <h3><b>React To Do List</b></h3>
+  <h3><b>React To Do List - Microverse Project Module 3</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+
+- [🧑🏻‍💻 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Available Scripts](#available-scripts)
+  - [Clone](#clone)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 React To Do List - Microverse Project Module 3<a name="about-project"></a>
+
+> Math Magicians React App developed with Nodejs, Html, Css, Js, Webpack, React
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://en.wikipedia.org/wiki/HTML">Html</a></li>
+    <li><a href="https://en.wikipedia.org/wiki/CSS">Css</a></li>
+    <li><a href="https://en.wikipedia.org/wiki/JavaScript">Vanilla Javascript</a></li>
+    <li><a href="https://webpack.js.org/">Webpack</a></li>
+    <li><a href="https://react.org/">React</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Add Task**
+- **Edit Task**
+- **Delete Task**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🧑🏻‍💻 Live Demo <a name="live-demo"></a>
+
+coming soon...
+<br/>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and use as a template for your own portafolio site, follow these steps.
+
+### Clone <a name="clone"></a>
+
+Clone this repository to your desired folder:
+
+```sh
+  https://github.com/chelobotix/Math-Magicians-Chelobotix.git
+```
+
+## 🤖 Available Scripts <a name="available-scripts"></a>
+
+This project use a Vite REact Template:
+
+### `npm create vite@latest`
+
+In the project directory, you can run:
+
+### `npm i`
+
+Runs the app in the development mode.\
+Open [http://localhost:5173](http://localhost:5173) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `npm run dev`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+### `npm run eject`
+
+**Note: this is a one-way operation. Once you `eject`, you can't go back!**
```

Comment:
Similar to the `npm test` command, this one also generates an error for the same reason. Also even if it was working,  it would be good you include what the command does. The warning isn't decriptive enough.